docs/lib/providers.ts

```diff
@@ -26,6 +26,7 @@ export default {
     params: { api_key: process.env.GATSBY_GEOCODEEARTH_API_KEY },
   }),
 
+  // TODO - need to update this - what's the ideal way?
```

Comment:
@kstratis I believe that the provider here is used in docs/components/Search.tsx and docs/components/Map.tsx, correct? We'll need to update the instantiation there as well.

Reply:
Sure! It's already been dealt with in the final version!